.gitignore

```diff
@@ -31,3 +31,4 @@ builds
 /flecsi/tutorial/*/ragged-data
 /flecsi/tutorial/*/forall
 /flecsi/tutorial/*/global-objects
+/.vs
```

Comment:
What in the FleCSI build system generates this file?

Reply:
It's being generated by my build system setup. I can remove this change.

Reply:
OK.  (Maybe you want to put it in `.git/info/exclude`?)